.github/workflows/MSDO.yml

```diff
@@ -0,0 +1,31 @@
+name: MSDO
```

Comment:
What does MSDO stand for? Microsoft Security DevOps Analysis? maybe then MSDA? MSDOA

Reply:
**M**icrosoft **S**ecurity **D**ev**O**ps

That I took from an example workflow and action usage. MSDO is the new cool kid on the block 😎